src/rhvqe/CreateNewJob.groovy

```diff
@@ -27,11 +27,11 @@ def createNewJob(args)
     def jobClusure = {
         _args ->
         def _jobMailBody =  "Scheduling ${_args.jobName} flow, \nplease follow ${_args.jobCurrentBuildUrl} to get more info."
-        notification.send_notification(
-          _args.jobMailSubject,
-          _jobMailBody,
-          _args.jobVersion
-        )
+//          notification.send_notification(
```

Comment:
If you don't want to execute this code - just delete it - don't comment it out.

Reply:
It is only for debugging. After finishing, I will return it back